custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml

```diff
@@ -36,7 +36,7 @@ subjects:
   name: custom-metrics-stackdriver-adapter
   namespace: custom-metrics
 ---
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.uthorization.k8s.io/v1
```

Comment:
typo

Reply:
This has been fixed: https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml#L39